build

```diff
@@ -2,6 +2,8 @@
 
 ORG_PATH="github.com/coreos"
 REPO_PATH="${ORG_PATH}/fleet"
+VERSION=$(git describe --dirty)
+GLDFLAGS="-X github.com/coreos/fleet/version.Version \"${VERSION}\""
```

Comment:
where did we end up on the space vs equals? https://github.com/coreos/ignition/blob/master/build#L7